client/src/Components/About/About.js

```diff
@@ -22,45 +22,52 @@ const About = () => {
   const user = useUser();
 
   const formatSection = (section, invertOrder) => {
-    const { title, paragraph, button } = section;
+    const { title, image, paragraph, button } = section;
 
     const getGridItem = (key, content) => {
       return (
-        <Grid item key={key} className={style.center} sm={6} xs={12}>
+        <Grid item key={key} sm={6} xs={12}>
           {content}
         </Grid>
       );
     };
 
     const titleGridItem = getGridItem(
-      title,
-      <Typography variant="h3" color="primary">
-        {title}
-      </Typography>
+      image,
+      <div className={style.imageWrapper}>
+        <img className={style.image} src={image} alt={title} />
+      </div>
     );
```

Comment:
If we handle themeng as well , we will have to use 8 svg imports here and will have to add more logic right. So thought it would be better to use this way of looping images instead. Please provide your valuable thoughts on this.

Reply:
its i usual UI pattern to show titles with paragraphs rather than with images right ? Please confirm to update as you suggested( image plus title)

Reply:
> If we handle themeng as well , we will have to use 8 svg imports here and will have to add more logic right. So thought it would be better to use this way of looping images instead. Please provide your valuable thoughts on this.

Yes, that's right. I think SVGs can be dynamically modified, so maybe it's worth it to look into that.

> its a usual UI pattern to show titles with paragraphs rather than with images right ? Please confirm to update as you suggested( image plus title)

Ok, let's keep them with the text then!